src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoApp.scala

```diff
@@ -86,18 +86,24 @@ trait EasyAuthInfoApp extends AutoCloseable with DebugEnhancedLogging with Appli
     for {
       ddm <- bagStore.loadDDM(bagId)
       ddmProfile <- getTag(ddm, "profile", bagId)
+      dcmiMetadata <- getOptionalTag(ddm, "dcmiMetadata")
       dateAvailable <- getTag(ddmProfile, "available", bagId).map(_.text)
       rights <- FileRights.get(ddmProfile, fileNode)
+      license <- configuration.licenses.getLicense(dcmiMetadata, rights)
       bagInfo <- bagStore.loadBagInfo(bagId)
       owner <- getDepositor(bagInfo, bagId)
-    } yield FileItem(bagId, path, owner, rights, dateAvailable)
+    } yield FileItem(bagId, path, owner, rights, dateAvailable, license)
   }
 
   private def getTag(node: Node, tag: String, bagId: UUID): Try[Node] = {
     Try { (node \ tag).head }
       .recoverWith { case _ => Failure(InvalidBagException(s"<ddm:$tag> not found in $bagId/dataset.xml")) }
   }
 
+  private def getOptionalTag(node: Node, tag: String): Try[Option[Node]] = Try {
```

Comment:
slight improvement: there is nothing that can go wrong here, so `Try` is not necessary. In that case, change the `<-` in the for-comprehension to a `=`.

```suggestion
  private def getOptionalTag(node: Node, tag: String): Option[Node] = {
```